packages/tailwind-config/src/hive-colors.ts

```diff
@@ -0,0 +1,42 @@
+const hiveYellow = '#E1FF00';
```

Comment:
This file should be exported? If yes `exports` field should be updated in package.json

Reply:
I'm relying on your `satisfies Config` in the Tailwind config, so we can access the colors from `theme.extend.colors`.

On the way I had an issue where `typesVersion` has overwritten the types, so I moved references to the `.d.ts` files to the `exports`. https://github.com/the-guild-org/shared-config/pull/569/commits/e01f33723d874d39ff15f7e4cd73ee8a56fc6b4c